pkg/chartverifier/certificate.go

```diff
@@ -16,76 +16,88 @@
 
 package chartverifier
 
-import "strconv"
+import (
+	"github.com/redhat-certification/chart-verifier/pkg/chartverifier/checks"
+	helmchart "helm.sh/helm/v3/pkg/chart"
+)
 
-type chartMetadata struct {
-	Name    string `json:"name" yaml:"name"`
-	Version string `json:"version" yaml:"version"`
-}
+var CertificateApiVersion = "v1"
```

Comment:
is this not actually `ReportApiVersion`? The tool does not generate certificates, it runs checks and report them, IMHO? BTW, I would extract Report type and corresponding types to a separate package, i.e. `pkg/chartverifier/api`?

Reply:
Use of certificate instead of report goes way beyond this PR. Have opened a new issue for that work: https://github.com/redhat-certification/chart-verifier/issues/78